docs/testing/stencil-testrunner/_category_.json

```
@@ -0,0 +1,5 @@
+{
+    "label": "Stencil Testrunner",
```

Should 'Testrunner' be two words for consistency with the rest of the doc?
```suggestion
    "label": "Stencil Test Runner",
```

---

fixed in https://github.com/ionic-team/stencil-site/pull/1344/commits/56678eaaacc5732782a12a9269d4c0500ed6bde8